src/renderer/contexts/GlobalNodeState.tsx

```diff
@@ -755,6 +761,29 @@ export const GlobalProvider = memo(
             [modifyNode, schemata]
         );
 
+        const useInputSize = useCallback(
+            (
+                id: string,
+                inputId: InputId,
+                inputSize?: InputSize
+            ): readonly [Size, (size: Size) => void, () => void] => {
+                const currentSize = inputSize ? inputSize[inputId] : { width: 0, height: 0 };
```

Comment:
This is incorrect. `inputSize` might not have a key `inputId`, so `currentSize` can be undefined.
```suggestion
                const currentSize = inputSize?.[inputId] ?? { width: 0, height: 0 };
```